great_expectations/data_context/store/checkpoint_store.py

```diff
@@ -104,9 +104,9 @@ def delete_checkpoint(
         name: Optional[str] = None,
         ge_cloud_id: Optional[str] = None,
     ) -> None:
-        key: Union[GeCloudIdentifier, ConfigurationIdentifier] = self._determine_key(
-            name=name, ge_cloud_id=ge_cloud_id
-        )
+        key: Union[
+            GeCloudIdentifier, ConfigurationIdentifier
+        ] = CheckpointStore.determine_key(name=name, ge_cloud_id=ge_cloud_id)
```

Comment:
@Shinnnyshinshin How come it is not `self.determine_key()`?..

Reply:
This was a mis-understanding on my part. Changed back to `self.`